app/components/offline_indicator/offline_indicator.js

```diff
@@ -23,6 +24,13 @@ const HEIGHT = 38;
 const OFFLINE = 'offline';
 const CONNECTING = 'connecting';
 const CONNECTED = 'connected';
+const IOSX_TOP_PORTRAIT = 88;
```

Comment:
Should we add this to ViewTypes constants?

Reply:
I think we should only add it if we plan on using it elsewhere. No need to clutter that file up with constants that we're only using in one place